README.md

```diff
@@ -83,10 +84,198 @@ the build script. Then, run the test script in the root directory. It
+
+### PT-WTE/EDS
+Refer to the scripts under peptidesim/inputs/pte/ as example
+1. Preparation
+
+Create a Python script for the preparation step. It is almost identical to the regular simulation. (Check the part1.py)
+
+If you wish to add EDS, you will need additional code for the preparation. Check part1.py as well as https://www.plumed.org/doc-v2.5/user-doc/html/_m_e_t_a_d.html.
+
+2. PT-WTE
+
+Create a second script and import all required modules. 
+Specify input. Note that the name of the job must be consistant with that in part 1.
+
+```python
+name = nameinpart1
+pickle_name = name + '.pickle'
+MPI_NP = 16
+peptide_cpoies = 1 #number of peptide per replica
+replicas = 16 #number of replicas
+remd_exhcange_period = 250
+#reload
+ps=3#initialize
+if(os.path.exists(pickle_name)):
+    print 'loading restart'
+    with open(pickle_name, 'r') as f:
+        ps = pickle.load(f)
+        #ps.pickle_name=pickle_name
+        print os.getcwd()
+        ps.rel_dir_name='.'
+```
+
+Write a function for getting replica exchange efficiency, this will be used in determining whether to end the simulation
+```python
+def get_replex_e(ps, replica_number):
+    with open(ps.sims[-1].location + '/' + ps.sims[-1].metadata['md-log']) as f:
+        p1 = re.compile('Repl  average probabilities:')
+        p2 = re.compile('Repl\s*' + ''.join(['([0-9\.]+\s*)' for _ in range(replica_number - 1)]) + '$')
+        ready = False
+        answer=-1
+        for line in f.readlines():
+            if not ready and p1.findall(line):
+                ready = True
+            elif ready:
+                match = p2.match(line)
+                
+                if match:
+                    answer= [float(s) for s in match.groups()]
+                    break
+            
+        return answer
+```
+
+Specify PTE inputs and generate plumed file
+```python
+pte_result = ps.pte_replica(mpi_np=MPI_NP, max_tries=3,min_iters=1, mdp_kwargs={'nsteps':int(400* 5*10**2), }, replicas=replicas,hills_file_location=os.getcwd(),hot=400,hill_height=0.6,sigma=250,bias_factor=16, eff_threshold=0.20,cold=278,exchange_period=200)
+pte_plumed_script=pte_result['plumed']
+replica_temps=pte_result['temperatures']
+with open('plumed_pte.dat', 'w') as f:
+    f.write(pte_plumed_script)
+ps.add_file('plumed_pte.dat')
+temps = ','.join(str(e) for e in replica_temps)
+kwargs = [ {'ref_t': ti} for ti in replica_temps]
+time_ns=5.0 
+replex_eff = 0.2
+for kw in kwargs:
+    kw['nsteps']=  int(time_ns * 5 * 10 ** 5)
+max_iterations=20
+min_iterations=10
+```
+
+If you are running EDS or require additional output from plumed, you can add additional plumed script by
+```python
+plumed_input0=textwrap.dedent(
+    '''
+    plumed script
+    ''')
+plumed_input=pte_plumed_script+plumed_input0
+with open('plumed_eds_conver_pt_wte_metad.dat', 'w') as f:
+    f.write(plumed_input)
+ps.add_file('plumed_eds_conver_pt_wte_metad.dat')
+```
```

Comment:
Why is this part of the file italicized?

Reply:
It was the problem with the html link that has "_". Fixed in new commit